[material-ui][Autocomplete] Remove autocomplete warning regarding value not equal to option (#43314)
mui/material-ui: docs/data/material/components/autocomplete/autocomplete.md

However, you can use different structures by providing a `getOptionLabel` prop.

> If your options are array of objects, also provide the `isOptionEqualToValue` prop to ensure correct selection and highlighting. By default, it uses strict equality between a option and value.

Reviewer: ```suggestion
If your options are an array of objects, you must provide the `isOptionEqualToValue` prop to ensure correct selection and highlighting. By default, it uses strict equality to compare options with the current value.
```

> If your options are an array of objects, you must provide the `isOptionEqualToValue` prop to ensure correct selection and highlighting. By default, it uses strict equality to compare options with the current value.

Reviewer 2: IMO this reads better:

```suggestion
If your options are objects, you must provide the `isOptionEqualToValue` prop to ensure correct selection and highlighting. By default, it uses strict equality to compare options with the current value.
```
